<?xml version="1.0" encoding="UTF-8"?><testrun name="CalculatorProjectTest" project="JunitProjectTest" tests="1" started="1" failures="1" errors="0" ignored="0">
  <testsuite name="io.cal.CalculatorProjectTest" time="0.055" displayname="CalculatorProjectTest" uniqueid="[engine:junit-jupiter]/[class:io.cal.CalculatorProjectTest]">
    <testcase name="positive2" classname="io.cal.CalculatorProjectTest" time="0.055" displayname="positive2()" uniqueid="[engine:junit-jupiter]/[class:io.cal.CalculatorProjectTest]/[method:positive2()]">
      <failure>
        <expected>true</expected>
        <actual>false</actual>
        org.opentest4j.AssertionFailedError: return false ==&gt; expected: &lt;true&gt; but was: &lt;false&gt;&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#13;
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)&#13;
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1156)&#13;
	at io.cal.CalculatorProjectTest.positive2(CalculatorProjectTest.java:68)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)&#13;
&#13;

      </failure>
    </testcase>
  </testsuite>
</testrun>
